src/synonyms.ts

```diff
@@ -0,0 +1,61 @@
+import * as _request from 'request-promise';
+import * as fs from 'fs';
+import * as Ajv from 'ajv';
+import { request } from 'https';
+
+
+export default async function synonyms({
+   payload }: { payload: { symbol: string } }) {
```

Comment:
If you pass in `fetch` as a variable here, you don't need to use proxyquire in the tests. 
Kind of like you do [here](https://github.com/repositive/metis/blob/faa671d3493fca33196d516f51883a7b7b545e6c/src/controllerSynonyms.ts#L14-L17)

Reply:
Can we discuss this bit? I'm not sure, if I get the whole idea.